### ISSUE TITLE
Set proper lang attribute on parent <html> element

### ISSUE THREAD

**Issue by CKFinderBOT** [ckfinder/ckfinder#186]
I was investigating how JAWS will read the following text and noticed that they are not read correctly unless I set the proper `lang` attribute, which makes sense ofc:

```
<kbd>Ctrl</kbd>. <kbd>CTRL</kbd>. <kbd>Control</kbd>. <kbd>CONTROL</kbd>.
<kbd>Strg</kbd>. <kbd>STRG</kbd>. <kbd>Steuerung</kbd>. <kbd>STEUERUNG</kbd>.
```

Btw. it looks like we are setting now `dir` property on `<body>`, we should do this on `<html>` tag instead because of the `<title>` element, which will be translatable soon. Also we should always set `dir`/`lang` attributes, not only e.g. if RTL language is chosen. That's because the parent page may be RTL, while CKFinder being unable to load a correct language file will fallback to LTR language (en).

**Comment by CKFinderBOT** [comment]
Fixed. The updated code will be publicly available in the upcoming release.
